ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -81,11 +81,11 @@ fi
 
 # multi-instance setup
 # shellcheck disable=SC2154
-param set MAV_SYS_ID $((1+px4_instance))
+param set MAV_SYS_ID $((px4_instance+1))
 simulator_udp_port=$((14560+px4_instance))
-udp_offboard_port_local=$((14557+px4_instance))
```

Comment:
This one needs to be different: 14550 is used for a GCS. Maybe 14580?

Reply:
done!